int64 drone_id
int64 marker_type
geometry_msgs/Point origin
---
bool success
